include/queryfunctions.php

```diff
@@ -179,6 +179,19 @@ function GetSetsForWorkout($workoutId){
         return $sets;
     };
 
+    function GetSetsAndMovementsForWorkout($workoutId){ // gets all movement, set, and instance info for each set
+        $movementsAndSets = dbQuery("
+        SELECT * FROM movementInstances INNER JOIN movements 
+        ON movementInstances.movementId = movements.movementId
+        INNER JOIN sets 
+        ON movementInstances.instanceId = sets.instanceId
+        WHERE movementInstances.workoutId = ".$workoutId."
```

Comment:
Technically should escape the workoutid instead of directly using it in your query